client/client-multi/client-c/toolchains/cc32xx.cmake

```diff
@@ -0,0 +1,43 @@
+set(cross_compiler_tools "/home/karnil/work/arm-toolchain/gcc-arm-none-eabi-4_9-2015q2")
+set(cc32xx_sdk "/home/karnil/work/cc3200/sdk")
+
+set(CMAKE_C_COMPILER   "${cross_compiler_tools}/bin/arm-none-eabi-gcc")
+set(CMAKE_CXX_COMPILER "${cross_compiler_tools}/bin/arm-none-eabi-g++")
+set(CMAKE_AR           "${cross_compiler_tools}/bin/arm-none-eabi-ar")
+set(CMAKE_LINKER       "${cross_compiler_tools}/bin/arm-none-eabi-ld")
+set(CMAKE_OBJCOPY      "${cross_compiler_tools}/bin/arm-none-eabi-objcopy")
+set(CMAKE_OBJDUMP      "${cross_compiler_tools}/bin/arm-none-eabi-objdump")
+set(CMAKE_NM           "${cross_compiler_tools}/bin/arm-none-eabi-nm")
+set(CMAKE_RANLIB       "${cross_compiler_tools}/bin/arm-none-eabi-ranlib")
+set(CMAKE_STRIP        "${cross_compiler_tools}/bin/arm-none-eabi-strip")
+
+set(CMAKE_C_FLAGS "-mthumb -mcpu=cortex-m4 -ffunction-sections -fdata-sections -MD -std=c99 -g -O0 -static"  CACHE STRING "" FORCE)
+set(CMAKE_A_FLAGS "-mthumb -mcpu=cortex-m4 -MD -static"  CACHE STRING "" FORCE)
+
+include_directories(${cc32xx_sdk}/inc)
```

Comment:
CMake toolchain file should not contain include directories and libraries. They should be defined in platform file.